Tests/SonarQube.Common.UnitTests/ProcessRunnerTests.cs

```diff
@@ -209,6 +208,44 @@ public void ProcRunner_MissingExe()
             logger.AssertSingleErrorExists("missingExe.foo");
         }
 
+        [TestMethod]
+        public void ProcRunner_ArgumentQuoting()
```

Comment:
New test to check that arguments passed to the launched process were correctly quoted.